project.mml

```diff
@@ -522,9 +522,12 @@ Layer:
             ELSE NULL
           END AS cycleway_right_oneway,
           CASE
+            WHEN bicycle IN ('no', 'private') THEN 'no'
```

Comment:
Minor comment, but might be better to merge into a `no_or_private` value or something like this? (for readability)

Reply:
can_bicycle is already a computed variable